app.py

```
@@ -149,6 +150,44 @@
+                    "network_timeout": 10.0,
+                    "timeout": 10.0,
+                    "keepalive_idle": 10,
+                    "keepalive_interval": 5,
+                    "keepalive_probes": 3,
+                    "allow_tls_fallback": False,
+                },
+            ),
+            max=app.config.get("LDAP_POOL_MAX", 50),
+        )
+    except ValueError as ldaperr:
+        logger.error(f"LDAP missing parameter {ldaperr}")
```

Ruff (0.14.8)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)